### ISSUE TITLE
False positive: `×  12:1  ToC item "Bridges" link "#Bridges" not found        remark-lint:awesome-toc`

### ISSUE THREAD

**Issue by Polycarbohydrate** [sindresorhus/awesome-lint#197]
The repository where this issue occurs: https://github.com/polycarbohydrate/awesome-tor

The issue:
```
×  12:1  ToC item "Bridges" link "#Bridges" not found        remark-lint:awesome-toc
```

I think this is a false positive as the link to the header of the `Bridges` section is usable and is a valid path.

**Comment by sindresorhus** [comment]
Linkified slugs are usually lowercased, so this may actually be valid.

**Comment by sindresorhus** [comment]
Yeah: https://github.com/Polycarbohydrate/awesome-tor/commit/0fdec97e51f8657d0583ce4723a513d072a2216a
